handlers/main.yml

```diff
@@ -1,9 +1,21 @@
 ---
+# The order here matters
 - name: restart telegraf
-  service: 
+  service:
     name: telegraf
     state: restarted
 
 - name: pause
   pause:
-    seconds: 3
\ No newline at end of file
+    seconds: 3
```

Comment:
This was already here, but the one I removed from was set to 6 seconds.  Not sure the reason for 3 vs. 6, should this be a property?